x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_requests_table.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiBasicTable, EuiEmptyPrompt, EuiTableFieldDataColumnType } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { CrawlerOverviewLogic } from '../crawler_overview_logic';
+import { CrawlRequest, readableCrawlerStatuses } from '../types';
+
+import { CustomFormattedTimestamp } from './custom_formatted_timestamp';
+
+const columns: Array<EuiTableFieldDataColumnType<CrawlRequest>> = [
+  {
+    field: 'id',
+    name: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.crawler.crawlRequestsTable.column.domainURL',
+      {
+        defaultMessage: 'Request ID',
+      }
+    ),
+  },
+  {
+    field: 'createdAt',
+    name: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.crawler.crawlRequestsTable.column.created',
+      {
+        defaultMessage: 'Created',
+      }
+    ),
+    render: (createdAt: CrawlRequest['createdAt']) => (
+      <CustomFormattedTimestamp timestamp={createdAt} />
+    ),
+  },
+  {
+    field: 'status',
+    name: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.crawler.crawlRequestsTable.column.status',
+      {
+        defaultMessage: 'Status',
+      }
+    ),
+    render: (status: CrawlRequest['status']) => readableCrawlerStatuses[status],
+  },
+];
+
+export const CrawlRequestsTable: React.FC = () => {
+  const { crawlRequests } = useValues(CrawlerOverviewLogic);
+
+  if (crawlRequests.length === 0) {
+    return (
+      <EuiEmptyPrompt
+        iconType="tableDensityExpanded"
+        title={
+          <h3>
```

Comment:
I can't spin up up Kibana right now to check, but can you confirm for me that axe-devtools doesn't throw an error for this heading level usage? I'm seeing a lot of `<h3>`s in this diff but not any `<h2>`s which makes me wonder

Reply:
Fixed in 6c13333

Reply:
Thanks a billion!